docs/kubernetes/KubernetesPolicy.md

```diff
@@ -0,0 +1,63 @@
+# Programming Calico Policy in Kubernetes
+The [Calico Kubernetes plugin](https://github.com/projectcalico/calico-docker/blob/master/docs/kubernetes/KubernetesIntegration.md) allows you to specify networking policy in the Kubernetes API using pod annotations. 
+
+## Prerequisites
+* A Kubernetes Deployment
+    - To implement service policy using recent builds of Kubernetes, make sure you have configured your kube-proxy using the `--legacy-userspace-proxy=false` option
+* v0.1.0+ of the Calico Kubernetes Plugin
```

Comment:
I'd link to the GitHub releases page here - its a bit confusing, since there are some hosted version of calico_kubernetes on calico-docker with version 0.4.* and 0.5.*, which are actually older than the 0.1.0 release on calico-kubernetes.